docs/design/chart_format.md

```diff
@@ -394,7 +370,7 @@ The following URI schemes must be supported:
 
 
 #### Special Form Google Storage and S3
-The [Google Storage scheme](https://cloud.google.com/storage/docs/interoperability) (`gs://`)  and the S3 scheme (`s3://`) can also be support. These forms do not follow the URL standard (bucket name is placed in the host field), but their behavior is well-documented.
+The [Google Storage scheme](https://cloud.google.com/storage/docs/interoperability) (`gs://`)  and the S3 scheme (`s3://`) can also be supported. These forms do not follow the URL standard (bucket name is placed in the host field), but their behavior is well-documented.
 
 
 Examples:
```

Comment:
In Appendix A below, the Personas are not immediately relevant to this document. Suggest that we change them to:
* Chart Developer: Responsible for building new charts.
* App Developer: Responsible for building apps that use charts.